### ISSUE TITLE
Layer - Navigation

### ISSUE THREAD

**Issue by cdespommare** [adeo/mozaic-design-system#1414]
### I want to propose an evolution/improvement on

Component

### Description

Hello Mozaïc, I would a request regarding the layer component on Mozaïc Freemarker.

For now, the header is static and allows just the change of the title. I have to integrate a layer, used and opened in several manners, but this time in a navigation layer itself. This implicates to have a left arrow to navigate to the previous layer, and a cross at the right to close all the layers opened (so with a configurable event).

I now this could be considered as a unique necessary case, but maybe this evolution could be useful to others.

### Describe the solution you would like

A left arrow to navigate to the previous layer, and a configurable cross at the right to close all the open layers

### Describe the alternatives you considered

Without it, I would have to do it manually (like it is done manually on the main mobile navigation for leroymerlin.fr)

### Additional comments

Example of an existing navigation layer (but a custom one made by the navigation team): ![image](https://user-images.githubusercontent.com/97841324/231496556-33d30393-f812-42b7-bbef-76e675065895.png)

**Comment by tiloyi** [comment]
The ball is in Pauline Lapierre's & Benjamin Dreux's court.
We are waiting for them to come back, to see how this component should evolve.

**Comment by cdespommare** [comment]
Hello @tiloyi 
Are they any news about this subject?

**Comment by tiloyi** [comment]
Hi @caqueste,
Is this planned for V2?

**Comment by caqueste** [comment]
@tiloyi Yes, the rework of the layer will include a navigation but quite different from the exemple. So the team in charge of the navigation will have to adapt the content to the new structure of the layer.
